### ISSUE TITLE
Request 

### ISSUE THREAD

**Issue by KoshikKumar17** [saminsumesh/TeamSakura#3]
As there is file store feature...
So please add one more command to delete files or batch files for which shareable link has been generated for admins...
Please @saminsumesh

**Comment by saminsumesh** [comment]
Yeah for sure @KoshikKumar17 , Just wait for futher Updates on this repo

**Comment by KoshikKumar17** [comment]
@saminsumesh Have you added

**Comment by KoshikKumar17** [comment]
@saminsumesh @belugaop do something https://github.com/saminsumesh/TeamSakura/issues/3#issue-1124170896
